### PR TITLE
test(storybook): add e2e test for storybook docs

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

[Test Scenario]: Storybook Docs pages #8016

### Description

This PR adds e2e tests for the storybook docs tab - making sure there are no errors thrown.

### Changelog

**New**

- tests for React and WC storybook docs

### CODE REVIEW

**Review by ariellalgilmore — APPROVED** [review]
LGTM!

File: packages/web-components/tests/e2e-storybook/cypress/integration/storybook-docs/storybook-docs.e2e.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     */
+    const components = cy.get('a[id*="explorercomponents"]');
+
+    components.each($component => {
+      const docsPath = $component.prop('href');
+
+      if (docsPath) {
+        const url = docsPath.split('?')[1].replace('path=/story', 'path=/docs');
+
+        cy.visit(`?${url}`);
```
Comment (jeffchew) [inline packages/web-components/tests/e2e-storybook/cypress/integration/storybook-docs/storybook-docs.e2e.js:25]: @annawen1 after messing around with Cypress a bunch, running a `cy.visit` multiple times in a single `it` test can cause some pretty bad memory issues. You should do an iteration and create multiple `it` tests instead.

**Review by jeffchew — CHANGES_REQUESTED** [review]

File: packages/web-components/tests/e2e-storybook/cypress/integration/storybook-docs/storybook-docs.e2e.js (hunk 1 above)
Comment (jeffchew) [reply packages/web-components/tests/e2e-storybook/cypress/integration/storybook-docs/storybook-docs.e2e.js:25]: Case in point, Github Actions is pretty unhappy with this test:

image

Comment (annawen1) [reply packages/web-components/tests/e2e-storybook/cypress/integration/storybook-docs/storybook-docs.e2e.js]: @jeffchew I ended up creating two tests, one to scrape all components from storybook and writing to the `fixtures/components.json` and then another one to import from that fixtures file to generate the tests. Looks like it's all passing! 🎆

**Comment by jeffchew** [comment]
@annawen1 looks good! I think we can omit the snapshots though. They don't looks like they would be too useful anyway:

image

Checking if there are any errors coming might be enough I think.

**Comment by annawen1** [comment]
> @annawen1 looks good! I think we can omit the snapshots though. They don't look like they would be too useful anyway:
> 
> image
> 
> Checking if there are any errors coming might be enough I think.


ok! I removed the snapshot commands from the tests!

**Review by jeffchew — APPROVED** [review]
LGTM!

**Pull request merged**